pages/en/irma-start.md

```diff
@@ -46,9 +46,8 @@ or tablet via the Android Playstore or the Apple App store.
 
 #### 1.2. Registration
 
-When you open the IRMA app for the first time, you can open an IRMA
-account via the start screens. When you do so, you are asked to
-provide:
+When you open the IRMA app for the first time, you have to set it up.
```

Comment:
General comment on the comments:
Feel free to ignore them. I'm not enthusiastic about the phrasing here and there but I'm not sure if the suggestions make it better.

Reply:
If you choose to change bits: I've only commented the English version, not the Dutch one but the comments would be similar there.